src/js/views/forms/form/form.scss

```diff
@@ -32,7 +32,7 @@
 .form__title {
   display: flex;
   font-size: 24px;
-  line-height: 1.2;
+  line-height: 1.4;
```

Comment:
Better aligns the title text with the form controls section.

![Screenshot 2023-03-20 at 3 34 19 PM](https://user-images.githubusercontent.com/35355575/226465541-294cc4c3-3712-4672-be30-1ade0b66c557.png)